### PR TITLE
Move to SSL

### PR DESCRIPTION
**Pull request by stefanor**
No SSL? Seriously? In this day and age?

Fortunately, that's trivial to fix.

### CODE REVIEW

**Comment by adrianmoisey** [comment]
This needs an accompanying chef PR that adds certificates to the `graphite` role

File: deploy/templates/apache2/vhost.conf.template [hunk 1]
```diff
@@ -1,10 +1,18 @@
-<VirtualHost *:80>
+<VirtualHost *:443>
     ServerName {{aconf.domain}}
 
     {{if 'alias_domains' in aconf }}
     ServerAlias {{for a in aconf.alias_domains}} {{a}} {{endfor}}
     {{endif}}
 
+    SSLEngine on
+    SSLCertificateFile {{ aconf.ssl.cert }}
```
Comment (adrianmoisey) [inline deploy/templates/apache2/vhost.conf.template]: Weird inconsistent whitespace inside the `{{ }}`

File: deploy/templates/apache2/vhost.conf.template [hunk 2]
```diff
@@ -38,4 +46,18 @@
...
+    {{if 'alias_domains' in aconf }}
+    ServerAlias {{for a in aconf.alias_domains}} {{a}} {{endfor}}
+    {{endif}}
+
+    Header add Yola-ID "{{aconf.domain}} %D %t"
+
+    RewriteEngine On
+    RewriteRule (.*) https://{{ aconf.domain }}%{REQUEST_URI}
+
+    CustomLog /var/log/apache2/graphite-access.log combined
```
Comment (adrianmoisey) [inline deploy/templates/apache2/vhost.conf.template]: Can you also add logging to the HTTPS vhost?

Comment (adrianmoisey) [inline deploy/templates/apache2/vhost.conf.template]: Oh, it's there. Github collapsed it for me.

**Comment by stefanor** [comment]
> This needs an accompanying chef PR that adds certificates to the graphite role

That role was rubbish. yola/chef#1407

File: deploy/templates/apache2/vhost.conf.template [hunk 3]
```diff
@@ -38,4 +46,18 @@
         Allow from all
     </Directory>
 </VirtualHost>
+<VirtualHost *:80>
+    ServerName {{aconf.domain}}
 
+    {{if 'alias_domains' in aconf }}
```
Comment (adrianmoisey) [inline deploy/templates/apache2/vhost.conf.template]: More strange whitespace

File: deploy/templates/apache2/vhost.conf.template [hunk 4]
```diff
@@ -38,4 +46,18 @@
...
+    ServerAlias {{for a in aconf.alias_domains}} {{a}} {{endfor}}
+    {{endif}}
+
+    Header add Yola-ID "{{aconf.domain}} %D %t"
+
+    RewriteEngine On
+    RewriteRule (.*) https://{{ aconf.domain }}%{REQUEST_URI}
```
Comment (adrianmoisey) [inline deploy/templates/apache2/vhost.conf.template]: Here too

**Comment by adrianmoisey** [comment]
Besides the whitespace, :+1:

**Pull request merged**